recipes/pythesint/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pythesint" %}
+{% set version = "1.2.0" %}
+{% set sha256 = "457ae24b8ffcee7a03ff9515c1578aec81d2bf3dd7a599d7cafc6e146e441b65" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [not py27 or win]
+
+requirements:
+  build:
```

Comment:
I think only `python` and `setuptools` are `build` requirements as you are using `--single-version-externally-managed`

Reply:
Thanks I did not know how this worked, will fix it.

Reply:
My mistake. I didn't see the `setup_requires` in the `setup.py`. Stuff listed in there is actually required unlike the stuff in `install_requires`

Reply:
Hehe, just figured that out :)